code/src/Components/NoseQuestion.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+
+export const NoseQuestion = ({ sizeOfNose, setSizeOfNose }) => {
```

Comment:
Here I think you could have used an iteration, props and an array in the App.js to only have one block of code in this component rather than hardcoding the three radios buttons.